llvm/test/CodeGen/SPIRV/extensions/enable-all-extensions.ll

```diff
@@ -0,0 +1,7 @@
+; RUN: llc -O0 -mtriple=spirv32-unknown-unknown --spirv-extensions=all %s -o - | FileCheck %s
+
+define i6 @getConstantI6() {
+  ret i6 2
+}
+
+; CHECK: OpExtension "SPV_INTEL_arbitrary_precision_integers"
```

Comment:
Why is there a check for only one extension here? Thanks

Reply:
We do not emit `OpExtension` instructions for extensions which are not needed/used in the SPIR-V module. Here the function uses i6 type and so we emit `OpExtension "SPV_INTEL_arbitrary_precision_integers"`